spec/index.html

```diff
@@ -816,6 +816,27 @@ <h3>Content Negotiation of RDF Datasets</h3>
       the consumer is expected to use the <a data-lt="rdf dataset">RDF dataset's</a> default graph.</p>
 
     </section>
+
+    <section id="section-dataset-quad" class="informative">
+      <h3>Dataset as a set of quads</h3>
+
+      <p>A <dfn>quad</dfn> is a conceptual projection of a <a>triple</a> with
+        either the <a>default graph</a> or a <a>named graph</a>.
+        A <a>quad</a> is the generalization of a <a>triple</a>
+        along with an optional <a>graph name</a>,
+        indicating a named graph containing that <a>triple</a>.
+      </p>
+
+      <p>A <a>quad</a> can be represented as a tuple composed of <a>subject</a>, <a>predicate</a>, <a>object</a>,
```

Comment:
That works because when the text says "subject" it has to be subject of something - the triple.